AutoGrade/fixtures/initial_language.json

```diff
@@ -0,0 +1,122 @@
+[
+  {
+    "model": "AutoGrade.language",
+    "pk": 1,
+    "fields": {
+      "name": "Python 3 (Pytest)",
+      "moss_name": "python",
+      "compile_cmd": "python3 -m py_compile {:mainfile}",
```

Comment:
This file contains `compile_cmd` and `execute_cmd` for execution of code through subprocess. This file is automatically load to database through data migrations (available in this PR).